tests/ssid_set_snmp.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2017
+#
+# All rights reserved.
+#
+# This file is distributed under the Clear BSD license.
+# The full text can be found in LICENSE in the root directory.
+
+import os, datetime, pexpect
+import rootfs_boot
+from devices import board, wan, lan, prompt
+from lib.installers import install_snmp
+from cbnlib import now_short
```

Comment:
This is a public repo- we should not put things that depend on a private repo here.

That being said, now_short() can probably move to tests/lib/common.py?